GoogleForCreators/web-stories-wp
# Saved Templates: remove virtual scroll

## Context
Saved Templates layout in the editor is using virtual scroll. When applying a template to a story, the scrolling will get stuck on the same 6 or so items. This is preventing the user from being able to use or see all saved templates. 


## Summary
Now that users have been able to generate image blobs for saved templates, we can remove the virtual scroll component from templateList.


## Relevant Technical Choices
Removed `TemplateList` in favor of `DefaultTemplateList` since most functionality needed was already present in that component.  


## To-do
- [x] check if any other components can be remove along side `templateList`.
- [x] since keyboard nav is using the `+` icon when naving up the list, the template is not fully in view, fix that.


## User-facing changes
### Before: 
https://user-images.githubusercontent.com/1820266/156233311-fcea218c-393c-4633-b2bf-08eca9237eb9.mov

### After: 
![savedtemplates](https://user-images.githubusercontent.com/1820266/156233166-89e7a81d-fde3-4d72-8272-fb089815173e.gif)



## Testing Instructions






This PR can be tested by following these steps:

1. Regression, make sure saved templates layout works as before
2. Ensure bug no longer present: 
3. Scroll toward end of save templates list
4. Add template to story
5. Attempt to scroll back up; scrolling should be smooth. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10437
